### PR TITLE
doc: parameter tags[]

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket: https://jira.kisio.org/browse/NAVP-946

### CODE REVIEW

**Review by nonifier — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

File: documentation/slate/source/includes/apis.md
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: Bob and Bobette in the official doc??? Seems more professional to give actual possible tags.

File: documentation/slate/source/includes/apis.md
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: Same comment for Bob and Bobette. Even though they're nice and welcome in our tests, it doesn't fell like they're in the right place here.

**Review by mbounabi — APPROVED** [review]

**Comment by mbounabi** [comment]
Wait for the feature to be merged before merging the doc

**Review by kinnou02 — APPROVED** [review]

File: documentation/slate/source/includes/apis.md
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: I would put 'alert' in :gb: (unless it mentions a search giving results) : "alerts" (no final `e`)
And 'incident' singular (no final `s` ) to be consistent

File: documentation/slate/source/includes/apis.md
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: same here

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**
